docs/User-guide.rst

```diff
@@ -0,0 +1,73 @@
+**********
+User Guide
+**********
+
+Once you understand :ref:`how dedupe works <how-it-works-label>`, and you've taken
```

Comment:
i think this paragraph might need a different framing? the current sections are all about what to do if you have really big data or you are not getting accurate results. Maybe this needs more of a troubleshooting frame and not a "how to use" framing?

Reply:
Good idea, will do